### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by thomasconner**
#### Description
Refactor code.

### CODE REVIEW

File: src/middleware/src/storage/src/websql.js [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
+const size = 5 * 1000 * 1000; // Database size in bytes
 let dbCache = {};
 let isSupported;
-const SIZE = 5 * 1000 * 1000; // 5mb
 
-export default class WebSQL {
+class WebSQL {
   constructor(name = 'kinvey') {
+    if (isDefined(name) === false) {
+      throw new Error('A name is required to use the IndexedDB adapter.', name);
```
Comment (tejasranade) [inline src/middleware/src/storage/src/websql.js]: @thomasconner  this should say WebSQL, not IndexedDB.

File: src/middleware/src/http.js [hunk 2]
```diff
@@ -9,7 +10,7 @@ function browserDetect(ua) {
   const rChrome = /(chrome)\/([\w]+)/;
   const rFirefox = /(firefox)\/([\w.]+)/;
   const rIE = /(msie) ([\w.]+)/i;
-  const rOpera = /(opera)(?:.*version)?[ \/]([\w.]+)/;
+  const rOpera = /(opera)(?:.*version)?[ /]([\w.]+)/;
```
Comment (tejasranade) [inline src/middleware/src/http.js:13]: @thomasconner is this deliberate? If so, what prompted it?

File: src/middleware/src/storage/src/websql.js [hunk 3]
```diff
@@ -1,26 +1,33 @@
...
+
     this.name = name;
   }
 
   openDatabase() {
     let db = dbCache[this.name];
 
     if (!db) {
-      db = global.openDatabase(this.name, 1, '', SIZE);
+      db = global.openDatabase(this.name, 1, '', size);
```
Comment (tejasranade) [inline src/middleware/src/storage/src/websql.js]: @thomasconner  What is the third parameter? Can we provide a meaningful value for it?

**Review by tejasranade — CHANGES_REQUESTED** [review]

File: src/middleware/src/storage/src/websql.js (hunk 3 above)
Comment (thomasconner) [reply src/middleware/src/storage/src/websql.js]: Let me see if I can think of something useful

File: src/middleware/src/http.js (hunk 2 above)
Comment (thomasconner) [reply src/middleware/src/http.js:13]: Is the opera regex deliberate?

File: src/middleware/src/storage/src/websql.js (hunk 1 above)
Comment (thomasconner) [reply src/middleware/src/storage/src/websql.js]: Fixed

File: src/middleware/src/http.js (hunk 2 above)
Comment (thomasconner) [reply src/middleware/src/http.js:13]: @tejasranade Eslint was throwing an error "unnecessary escape character"

**Review by tejasranade — APPROVED** [review]

**Pull request merged**